servo/cli.py

```diff
@@ -1149,6 +1150,12 @@ def run(
                 "--no-poll",
                 help="Do not poll the Opsani API for commands",
             ),
+            no_diagnostics: Optional[bool] = typer.Option(
+                None,
+                "--no-diagnostics",
+                help="Do not poll the Opsani API for diagnostics",
+                envvar="NO_DIAGNOSTICS",
+            ),
```

Comment:
Please replace this with a new option on `servo.configuration.BaseServoConfiguration`. Analogous shell invocation would be `SERVO_NO_DIAGNOSTICS=True servo run ....`

Reply:
Ok, I think I've got everything in except this one - likely in need of an implementation hint here

Reply:
Add a new field here: https://github.com/opsani/servox/blob/138f762965073a865b1742f2c9e448d273e2eff3/servo/configuration.py#L491

```
no_diagnostics: bool = pydantic.Field(
    default=False, description="Do not poll the Opsani API for diagnostics"
)
```

It can then be referenced on the configuration of the servo `current_servo.config.no_diagnostics`

(also feel free to invert the name of this if you think it makes more sense `no_diagnostic` -> `diagnostics`)